### PR TITLE
Bug/finalise integration locations

### PR DESCRIPTION
**Pull request by connorpgpmcelroy**
### JIRA link (if applicable) ###
Indirectly concerns https://tools.hmcts.net/jira/browse/EUI-5193

### Change description ###
Integrate actual locations endpoint

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x ] No
```

### CODE REVIEW

**Review by udaydenduluri33 — APPROVED** [review]
@connorpgpmcelroy thanks for changes

File: api/locations/index.ts [hunk 1]
```diff
@@ -21,46 +22,61 @@ const url: string = getConfigValue(SERVICES_PRD_API_URL);
...
+      courtVenue.epimms_id === id
+    );
+    const mappedLocationModel = mapCourtVenuesToLocationModels(filteredResults);
+    res.send(mappedLocationModel).status(response.status);
   } catch (error) {
     next(error);
   }
 }
 
-function getCourtTypeIdsByService(serviceIdArray: string[]): string {
```
Comment (kuda1992) [inline api/locations/index.ts:55]: Can we not just remove these functions if there are not being used?

File: api/locations/index.ts [hunk 2]
```diff
@@ -21,46 +22,61 @@ const url: string = getConfigValue(SERVICES_PRD_API_URL);
...
   return '';
-}
+} */
 
-function concatCourtTypeWithoutDuplicates(array1: number[], array2: number[]) {
+/* function concatCourtTypeWithoutDuplicates(array1: number[], array2: number[]) {
   return array1.concat(array2.filter(item => array1.indexOf(item) < 0));
+} */
+
+function mapCourtVenuesToLocationModels(courtVenues: CourtVenue[]): CourtVenue {
```
Comment (kuda1992) [inline api/locations/index.ts:80]: And is this function necessary if it's just getting the first value in the array?

**Review by kuda1992 — CHANGES_REQUESTED** [review]
Minor comments but generally looks good.

File: api/locations/index.ts (hunk 1 above)
Comment (connorpgpmcelroy) [reply api/locations/index.ts:55]: I'll check if they are and remove if not

File: api/locations/index.ts (hunk 2 above)
Comment (connorpgpmcelroy) [reply api/locations/index.ts:80]: Will clarify this

**Review by kuda1992 — APPROVED** [review]
Thanks for making the changes

**Pull request merged**